rir/src/compiler/translations/rir_2_pir/rir_2_pir.cpp

```diff
@@ -200,20 +200,22 @@ bool Rir2Pir::compileBC(const BC& bc, Opcode* pos, Opcode* nextPos,
         break;
 
     case Opcode::ldvar_:
-    case Opcode::ldvar_for_update_:
+    case Opcode::ldvar_cache_:
+    case Opcode::ldvar_for_update_cache_:
```

Comment:
only saw this now, but in general cached (with d) would be a much better name for all of them. because cache is used as adverb and not verb: perform a lookup, a cached one.